code/modules/fishing/fishing_spots.dm

```diff
@@ -103,3 +148,108 @@ ABSTRACT_TYPE(/datum/fishing_spot)
 		if(!istype(., /obj/item/reagent_containers/food/snacks))
 			var/obj/machinery/deep_fryer/fryer = target
 			. = fryer.fryify(.)
+
+/datum/fishing_spot/toilet
+	fishing_atom_type = /obj/item/storage/toilet
+	fish_available = list(/obj/item/fish/carp = 40,\
+	/obj/item/fish/bass = 30,\
+	/obj/item/fish/salmon = 20,\
+	/obj/item/fish/herring = 15,\
+	/obj/item/fish/red_herring = 5,\
+	/obj/item/reagent_containers/food/snacks/burrito = 10,\
+	/obj/item/clothing/head/plunger = 3,\
+	/obj/item/reagent_containers/pill/cyberpunk = 3,\
+	/obj/item/reagent_containers/syringe/jenkem = 3,\
+	/obj/item/hairball/inert = 2,\
+	/obj/item/electronics/scanner = 2,\
+	/obj/item/skull = 2,\
+	/obj/item/clothing/gloves/ring/gold = 1,\
+	/obj/item/clothing/mask/niccage = 1,\
+	/obj/item/clothing/head/DONOTSTEAL = 1)
+
+/datum/fishing_spot/sink
+	fishing_atom_type = /obj/submachine/chef_sink
+	fish_available = list(/obj/item/fish/carp = 40,\
+	/obj/item/fish/bass = 30,\
+	/obj/item/fish/salmon = 20,\
+	/obj/item/fish/herring = 15,\
+	/obj/item/fish/red_herring = 5,\
+	/obj/item/coin = 5,\
+	/obj/item/reagent_containers/food/snacks/goldfish_cracker = 4,\
+	/obj/item/hairball/inert = 2,\
+	/obj/item/clothing/gloves/ring/gold = 1,\
+	/obj/critter/snake = 1,\
+	/obj/item/reagent_containers/food/snacks/condiment/custard = 1,\
+	/obj/item/reagent_containers/food/snacks/haggis = 1) //snake, custard, and haggis are nethack references, haggis is a meat "pudding"
+
+/datum/fishing_spot/pool
+	fishing_atom_type = /turf/simulated/pool
+	fish_available = list(/obj/item/fish/carp = 40,\
+	/obj/item/fish/bass = 30,\
+	/obj/item/fish/salmon = 20,\
+	/obj/item/fish/herring = 15,\
+	/obj/item/fish/red_herring = 5,\
+	/obj/item/beach_ball = 3,\
+	/obj/item/clothing/shoes/flippers = 3,\
+	/obj/item/clothing/gloves/water_wings = 3,\
+	/obj/item/inner_tube/random = 3,\
+	/obj/item/reagent_containers/food/drinks/curacao = 3,\
+	/obj/item/rubberduck = 2,\
+	/obj/item/organ/chest = 1,\
```

Comment:
We have an actual treasure chest in the code, but you choose to do this. 
smhing my head v_v

Reply:
I thought it was funnier